Merge_Sort_3_Way/Merge_Sort_3_Way.c

```diff
@@ -0,0 +1,123 @@
+#include<stdio.h>
+//#include<stdlib.h>
+
+void mergesort(long int *arr1, int p , int q ,long int *arr2);
+int merge(long int *arr1,int l,int m1, int m2, int h, long int *arr2 );
+
+int main()
+{ 
+
```

Comment:
Please remove unnecessary blank lines like these. Blank lines should only be added
- Between sections of a source file
- Between class and interface definitions
- Between methods
- Between the local variables in a method and its first statement
- Before a block or single-line comment
- Or between logical sections inside a method to improve readability